src/mscorlib/ref/mscorlib.cs

```diff
@@ -15,6 +15,12 @@ public static partial class EnvironmentAugments
         public static void FailFast(string message, System.Exception error) { }
         public static string[] GetCommandLineArgs() { throw null; }
     }
+
+    public static partial class AppDomainAugments
```

Comment:
Agreed.

UnhandledException could also go to AppContext which which is where we were thinking about exposing it also.